marvin-toolkit/entrypoint.sh

```
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# Build the test harness
+cd example/rust-crypto
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)